packages/sit-onyx/src/components/OnyxFileUpload/OnyxFileUpload.ct.tsx

```diff
@@ -268,3 +295,23 @@ test("should have hide button", async ({ mount, page }) => {
   await expect(hideButton).toBeHidden();
   await expect(revealButton).toBeVisible();
 });
+
+test("should show required error message", async ({ mount, page }) => {
+  // ARRANGE
+  const component = await mount(
+    <OnyxFileUpload required showError style={{ padding: "1rem", width: "32rem" }} />,
+  );
+
+  const button = component.getByRole("button", { name: "Click to select" });
+  const errorMessage = component.locator(".onyx-file-upload__required_error");
+
+  // ASSERT
+  await expect(errorMessage).toBeVisible();
+  await expect(errorMessage).toHaveText("You need to upload at least one file");
```

Comment:
(Optional:) To improve the type-safety and dependency on the i18n messages we can use the messages directly from the `en-US.json` file. 


```ts
import enUs from "../../i18n/locales/en-US.json" with { type: "json" };
```

```suggestion
  const button = component.getByRole("button", { name: enUs.fileUpload.clickToUpload });
  const errorMessage = component.locator(".onyx-file-upload__required_error");

  // ASSERT
  await expect(errorMessage).toBeVisible();
  await expect(errorMessage).toHaveText(enUs.fileUpload.requiredError);
```